spec/lib/hca/service_spec.rb

```diff
@@ -12,9 +12,9 @@
       root = Rails.root.join('spec', 'fixtures', 'hca', 'conformance')
       Dir[File.join(root, '*.json')].map { |f| File.basename(f, '.json') }.each do |form|
         it "properly formats #{form} for transmission" do
-          json = JSON.load(root.join("#{form}.json"))
+          json = JSON.parse(root.join("#{form}.json"))
```

Comment:
`load` reads and parses. Either undo this or add a `File.read` and specs should pass.